### PR TITLE
(DOCSP-4871): Added banner for MongoDB Agent.

### PR DESCRIPTION
**Pull request by atsansone**
@schmalliso : This [adds the banner](https://docs-staging.cloudmanager.mongodb.com/anthonysansone/DOCSP-4871/index.html) notifying the users that the Agents are combining into the single MongoDB Agent.

### CODE REVIEW

File: themes/mms-cloud/page.html [hunk 1]
```diff
@@ -42,7 +42,33 @@
...
+{%- block alertbar -%}
+    {%- if theme_has_mdb_agent %}
+        <div class="alert alert-info">
+          <span class="alert-message">
+
+            <h4 class="alert-heading">Change is coming soon to the MongoDB Agents!</h4>
+            Monitoring, Backup and Automation are merging to become
+            the MongoDB Agent.
+
+            <a class="btn btn-info" href="https://docs.cloudmanager.mongodb.com/reference/faq/faq-mongodb-agent/">Learn More</a>
```
Comment (jdestefano-mongo) [inline themes/mms-cloud/page.html]: I don't think the button looks good.

Can we do the usual: "For more information, see the MongoDB Agent FAQ<link>."

File: themes/mms-cloud/page.html [hunk 2]
```diff
@@ -42,7 +42,33 @@
...
 
-{%- block alertbar -%}{%- endblock -%}
+{%- block alertbar -%}
+    {%- if theme_has_mdb_agent %}
+        <div class="alert alert-info">
+          <span class="alert-message">
+
+            <h4 class="alert-heading">Change is coming soon to the MongoDB Agents!</h4>
+            Monitoring, Backup and Automation are merging to become
+            the MongoDB Agent.
```
Comment (jdestefano-mongo) [inline themes/mms-cloud/page.html]: Deferring to Melissa on copy. Don't merge without her signoff.

File: themes/mms-cloud/page.html [hunk 3]
```diff
@@ -42,7 +42,33 @@
 
 {%- block onboarding_modal -%}{%- endblock -%}
 
-{%- block alertbar -%}{%- endblock -%}
+{%- block alertbar -%}
+    {%- if theme_has_mdb_agent %}
+        <div class="alert alert-info">
```
Comment (jdestefano-mongo) [inline themes/mms-cloud/page.html]: Can we add a left margin of 20px to the alert info css so it lines up with the rest of the content on the page?

```
.alert-info {
    margin-bottom: 40px;
    margin-left: 20px;
}
```

Comment (atsansone) [reply themes/mms-cloud/page.html]: Agreed. Fixed.

File: themes/mms-cloud/page.html (hunk 2 above)
Comment (atsansone) [reply themes/mms-cloud/page.html]: Agreed.

File: themes/mms-cloud/page.html (hunk 1 above)
Comment (atsansone) [reply themes/mms-cloud/page.html]: Done.

File: themes/mms-cloud/page.html [hunk 4]
```diff
@@ -42,7 +42,33 @@
...
+
+            <h4 class="alert-heading">Change is coming soon to the MongoDB Agents!</h4>
+            Monitoring, Backup and Automation are merging to become
+            the MongoDB Agent.
+
+            <a class="btn btn-info" href="https://docs.cloudmanager.mongodb.com/reference/faq/faq-mongodb-agent/">Learn More</a>
+
+          </span>
+        </div>
+    {%- else %}
```
[the remark this answers is not in the document]
Comment (themantissa) [reply themes/mms-cloud/page.html]: This feels redundant on wording - Monitoring, Backup and Automation are merging to become _the MongoDB Agent_. Read the FAQ to learn about what's coming with _the MongoDB Agent_.  Can we rework the second sentence some to flow smoother?

**Comment by schmalliso** [comment]
@atsansone I'll be OOO Friday; if this comes back into your focus tomorrow and needs eyes before Monday, hit up Sophie or Dan 😺. My question re: the `else` still needs addressing.

**Comment by atsansone** [comment]
@schmalliso : The `else` was to prevent this from appearing anywhere but `master`, (i.e. Cloud Manager). I enabled it for our viewing in staging right now. I will probably remove it later.

**Comment by schmalliso** [comment]
@atsansone So you're not actually planning to have the content in the `if` also in the `else` block when we merge?

**Comment by atsansone** [comment]
@schmalliso : At this time, that's correct. I may need to see if I can make a better `if... else` so this appears in only certain releases (branches).

**Comment by schmalliso** [comment]
@atsansone You should be setting a `has_mdb_agent` variable to `true` in the conf.py of the branch that you want to display the alert. That way you can get rid of the `else` entirely and it'll work the way I believe you want.

For reference, we set `eol` to `true` in the v3.0 branch of the manual here: https://github.com/mongodb/docs/blob/v3.0/conf.py#L152
which causes the `if_eol` block in the manual's theme to be displayed:
https://github.com/mongodb/docs-tools/blob/master/themes/manual/page.html#L25

**Comment by atsansone** [comment]
@schmalliso : OK, I will do that! I will make that change. If I do that and remove this else, is this LGTM?

**Comment by schmalliso** [comment]
Squashed & merged.

**Pull request closed without merging**